src/Analysis/Ast/Impl/Types/PythonClassType.Generics.cs

```diff
@@ -127,7 +127,7 @@ internal partial class PythonClassType {
                     }
 
                     // Set specific class bases 
-                    classType.SetBases(specificBases.Concat(newBases), args.Eval.CurrentScope);
+                    classType.SetBases(specificBases.Concat(newBases), args.Eval?.CurrentScope);
```

Comment:
When does the `args.Eval` become `null`?

Reply:
When class is restored from the database, there is no evaluator. However, in the restore case there is no need to disambiguate bases and unroll loops since they were unrolled during analysis and before the persistence.

Reply:
Added comment